### PR TITLE
Refactor/primary secondary tertiary colours

### PR DESCRIPTION
**Pull request by sturobson**
- updates all themes so that:
  - primary is blue
  - secondary is greent
  - tertiary is a dark grey
  - quaternary is yellow
  - quinary is a light grey
- corrects instances of text colour on secondary - as white fails WCAG colour contrast
- adds 'theme colours' into the design tokens
- adds a 'theme' function
- introduce this into all components

- [x] all the changelogs

### CODE REVIEW

File: components/vf-design-tokens/src/maps/vf-colors.yml [hunk 1]
```diff
@@ -3,6 +3,39 @@ imports:
 
 props:
 
+  # EMBL Primary, Secondary, Tertiary Colours
```
Comment (khawkins98) [inline components/vf-design-tokens/src/maps/vf-colors.yml]: Hurrah!

**Comment by sturobson** [comment]
I'll do the changelog after moving this to ready for review

**Comment by khawkins98** [comment]
Looks good. Nudge on the changelog.

**Review by khawkins98 — APPROVED** [review]

**Comment by sturobson** [comment]
@khawkins98 I merged latest `develop` into this branch because how changelog files are now written.

Not sure if you want to take another look before this gets merged?

**Comment by sturobson** [comment]
This PR changed the foreground (text) colour when using the EMBL green as a background colour from white on green, to black on green.

This change was made so that aspects of the site that do this pass WCAG color contrast guidelines:

Screenshot 2020-09-02 at 22 01 58

Screenshot 2020-09-02 at 22 01 39

**Comment by khawkins98** [comment]
Good record, thanks @sturobson.

If this becomes a sticking point, it might be an opportunity to revise the EMBL green on web, or we could use one of the embl green tints. 

Or the [EMBL Green dark tint](https://wwwdev.embl.org/guidelines/design/brand-guidelines/colour/) (`#007a53`) https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=007A53

### MS green

For example MX/xbox have a similar but shade darker green (`#0F760F`)

https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=0F760F

image

### Github green

Aside, noticed the github "comment" button below fails: https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=2EA44F

image

### Parting words

I don't think any further action is needed, just leaving notes for "the future".

**Pull request merged**